src/containers/GraphRouteHandlerContainer.tsx

```diff
@@ -0,0 +1,34 @@
+import { connect } from 'react-redux';
```

Comment:
I think this is a problem, I think it basically undoes what I had done in the activeNamespace handling PR.  I don't think we want a container on GraphRouteHandler.  This is when the URL handling and the redux handling mix.  I think we want the URL handling to be "pure" and to always be truth.  Pass that truth to GraphPage and then handle all of the redux changes at that level.  This is what will give us bookmarkability, I think.   We'll have to talk about whether this is really necessary, I think.

Reply:
this PR is older than yours and should have gone in first, then you could have changed it in yours. Now I just keep rebasing and refactoring day after day.